lib/easypost/client_configuration.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
```

Comment:
thought: I don't see a need to break this all out into a separate class. Let's simply consolidate all Client logic into the Client class.

Reply:
Thought process: https://github.com/EasyPost/easypost-csharp/pull/446

Reply:
I just don't see the need or desire for the extra complexity. The client object is dead simple, let's keep it that way.

Reply:
We will consolidate this to Client